app/helpers/contact_helper.rb

```diff
@@ -0,0 +1,14 @@
+module ContactHelper
+  def contact_link(contact)
+    if contact.present?
+      text = safe_join([fa_icon('envelope'), ' ', contact.name])
+      link_to(text, contact_path(contact))
+    end
+  end
+
+  def contact_from_constant(constant:)
+    contact = Constant.find_by(name: constant).try(:value)
+    contact = Contact.find_by(id: contact)
```

Comment:
1. But I do not want it to raise an `ActiveRecord::RecordNotFound`.
2. I use two different models, could have named the first variable differently.

We have the model `Constant` where I want to add the current `id` of a `Contact` (to avoid hardcoding). Then I can look for the constant to get the id of the contact.